test/second_q/formats/fcidump/test_fcidump_dumper.py

```diff
@@ -0,0 +1,176 @@
+        self.num_electrons = None
+        self.spin_number = None
+        self.wf_symmetry = None
+        self.orb_symmetries = None
+        self.mo_onee = None
+        self.mo_eri = None
+
+    @abstractmethod
+    def subTest(self, msg, **kwargs):
+        # pylint: disable=invalid-name
+        """subtest"""
+        raise Exception("Abstract method")
+
+    @abstractmethod
+    def assertAlmostEqual(self, first, second, places=None, msg=None, delta=None):
+        """assert Almost Equal"""
+        raise Exception("Abstract method")
+
+    @abstractmethod
+    def assertEqual(self, first, second, msg=None):
+        """assert equal"""
+        raise Exception("Abstract method")
+
+    @abstractmethod
+    def assertSequenceEqual(self, seq1, seq2, msg=None, seq_type=None):
+        """assert Sequence Equal"""
+        raise Exception("Abstract method")
+
+    def test_dumped_inactive_energy(self):
+        """dumped inactive energy test"""
+        self.log.debug("Dumped inactive energy is %g", self.dumped["ECORE"])
+        self.assertAlmostEqual(self.dumped["ECORE"], self.core_energy, places=3)
+
+    def test_dumped_num_molecular_orbitals(self):
+        """dumped number of orbitals test"""
+        self.log.debug("Dumped number of orbitals is %d", self.dumped["NORB"])
+        self.assertEqual(self.dumped["NORB"], self.num_molecular_orbitals)
+
+    def test_dumped_num_electrons(self):
+        """dumped number of electrons test"""
+        self.log.debug("Dumped number of electrons is %d", self.dumped["NELEC"])
+        self.assertEqual(self.dumped["NELEC"], self.num_electrons)
+
+    def test_dumped_spin_number(self):
+        """dumped spin number test"""
+        self.log.debug("Dumped spin number is %d", self.dumped["MS2"])
+        self.assertEqual(self.dumped["MS2"], self.spin_number)
+
+    def test_dumped_wave_function_sym(self):
+        """dumped wave function symmetry test"""
+        self.log.debug("Dumped wave function symmetry is %d", self.dumped["ISYM"])
+        self.assertEqual(self.dumped["ISYM"], self.wf_symmetry)
+
+    def test_dumped_orbital_syms(self):
+        """dumped orbital symmetries test"""
+        self.log.debug("Dumped orbital symmetries is %s", self.dumped["ORBSYM"])
+        self.assertEqual(self.dumped["ORBSYM"], self.orb_symmetries)
+
+    def test_dumped_h1(self):
+        """dumped h1 integrals test"""
+        self.log.debug("Dumped h1 integrals are %s", self.dumped["H1"])
+        np.testing.assert_array_almost_equal(
+            np.absolute(self.dumped["H1"]), np.absolute(self.mo_onee), decimal=4
+        )
+
+    def test_dumped_h2(self):
+        """dumped h2 integrals test"""
+        self.log.debug("Dumped h2 integrals are %s", self.dumped["H2"])
+        np.testing.assert_array_almost_equal(
+            np.absolute(self.dumped["H2"]), np.absolute(self.mo_eri), decimal=4
+        )
+
+
+@unittest.skip("Until the FCIDump can handle non-beta spin cases")
```

Comment:
Huh.. I thought it should already be possible to do this :thinking: 
If this test really fails, let me know, then I can debug/fix this locally :+1: